server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -601,16 +579,6 @@ public void insert(InsertRowPlan insertRowPlan) throws StorageEngineException, M
 
     try {
       virtualStorageGroupProcessor.insert(insertRowPlan);
-      if (config.isEnableStatMonitor()) {
```

Comment:
It seems that we have to remove the parameter in the config file.

Reply:
ok I remove it.